FrontEnd/src/components/ReCaptcha/ReCartchaLoader.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+
+const ReCaptchaLoader = () => {
```

Comment:
Do we need it?  Does `react-google-recaptcha` handle it?

Reply:
You`re right, deleted redundanr code